packages/theme/src/baseTheme.ts

```diff
@@ -24,13 +24,23 @@ import {
   SimpleThemeOptions,
 } from './types';
 
+import { pageTheme } from './pageTheme';
+
 const DEFAULT_FONT_FAMILY =
   '"Helvetica Neue", Helvetica, Roboto, Arial, sans-serif';
 
 export function createThemeOptions(
   options: SimpleThemeOptions,
 ): BackstageThemeOptions {
-  const { palette, fontFamily = DEFAULT_FONT_FAMILY } = options;
+  const {
+    palette,
+    fontFamily = DEFAULT_FONT_FAMILY,
+    defaultPageTheme,
```

Comment:
We could allow `pageThemes` to be passed in here, and default it to the predefined ones.

Reply:
Confused! Should I remove ```defaultPageTheme``` option and then add ```pageTheme```? Or keep it as it and add.

Reply:
Keeping both,

```tsx
const {
  palette,
  fontFamily = DEFAULT_FONT_FAMILY,
  pageThemes = defaultPageThemes,
  defaultPageTheme,
} = options
```

That way it'll a bit more straightforward to do some simple overrides of the page theme.